dsbapi/__init__.py

```diff
@@ -7,20 +7,53 @@
 import uuid
 import base64
 
+DATA_URL = "https://app.dsbcontrol.de/JsonHandler.ashx/GetData"
```

Comment:
Sollte Konstante in Klasse selbst sein (Am besten im Code (weil nur einer und dann noch seit 2015 statischer API Endpoint) oder im Konstruktor